### DISCUSSION TITLE
shadowsocks multiple passwords on a single port

### DISCUSSION THREAD

**Discussion by 63110** [v2fly/v2ray-core#2645] [Q&A]
Hi. I have a question regarding multiple passwords on a single port. I saw that this feature was already implemented in ss. But what about V2FLY?

https://github.com/shadowsocks/shadowsocks-rust/commit/ac225f0f3651bc9f1b2d94a01713653b155349cc

Will this feature be implemented ? 

{
    "inbounds": [
        {
            "port": 1234, // server listening port
            "protocol": "shadowsocks",
            "settings": {
                "clients": [
                    {
                        "id": "71828b6842ea"
                    },
                    {
                        "id": "9587f5894284f"
                    }
                ]
            }
        }
    ],

**Comment by AkinoKaede** [comment]
I've tried to develop this feature, but couldn't find a good solution for compatibility with none cipher

https://github.com/AkinoKaede/v2ray-core/tree/dev-shadowsocks-multi-users

**Reply by 63110** [reply]
I think this is a very rare case, more often, wanted to see 1 port, and many passwords that would separate users, but don't propagate many ports
